src/praisonai/praisonai/runtime/__init__.py

```
@@ -1,36 +1,35 @@
-The thin CLI attaches to a running runtime when present (via a loopback
-HTTP transport described by a lockfile) and falls back to in-process
-execution when it is not. Nothing here is imported eagerly by the core
-SDK — it is wrapper-layer deployment/UX infrastructure.
+import praisonai_code.runtime as _runtime
+
+# Alias submodules so ``praisonai.runtime.<sub>`` resolves to the moved module.
+for _name in ("descriptor", "client", "server", "__main__"):
+    try:
+        _mod = __import__(f"praisonai_code.runtime.{_name}", fromlist=[_name])
+    except Exception:  # pragma: no cover - optional submodule
+        continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)